### PR TITLE
Gio/app UI refactor

### PR DESCRIPTION
**Pull request by digmel**
All components and screen  UI refactor, removing `react-native-paper` and `theme`

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,5 @@
 import React from 'react';
-
-import {Provider as PaperProvider} from 'react-native-paper';
-import Feather from 'react-native-vector-icons/Feather';
-import {ApolloClient, InMemoryCache, gql, ApolloProvider} from '@apollo/client';
-import {theme, navigationTheme} from '@styles';
+import {ApolloClient, InMemoryCache, ApolloProvider} from '@apollo/client';
```
Comment (lasharela) [inline src/App.tsx:2]: We have to move apollo in to it's own directory.

File: src/components/config/color.ts [hunk 2]
```diff
@@ -24,4 +24,6 @@ export const color = {
   google: '#DB4437',
   facebook: '#3B5998',
   apple: '#000000',
+
```
Comment (lasharela) [inline src/components/config/color.ts:27]: Don'w we have dark color?

File: src/components/header/Header.tsx [hunk 3]
```diff
@@ -45,17 +45,24 @@ export const Header: FC<THeaderProps> = ({
     <View
       style={[styles.container, containerDynamicStyle, containerStyle]}
       {...props}>
```
Comment (lasharela) [inline src/components/header/Header.tsx:47]: Doesn't do much to spread props here. Not needed I guess... we have style for this view and there is not much props left you can spread.

File: src/components/label/Label.tsx [hunk 4]
```diff
@@ -46,7 +46,7 @@ export const Label: FC<TLabelProps> = ({
       style={[styles.container, containerDynamicStyle, containerStyle]}
       {...props}>
```
Comment (lasharela) [inline src/components/label/Label.tsx:47]: Same here. We don't need to spread propr... would be good to spread for components were RN native element has his own proprs. for example ScrollView or Button and if we'll wrap with our component then to keep RN native props we have to spread.

File: src/components/shared/getSizeStyle.ts [hunk 5]
```diff
@@ -1,21 +1,21 @@
-import {size as configSize, EButtonSize} from '@components';
+import {size as configSize, TButtonSize} from '@components';
 import {StyleProp, ViewStyle, TextStyle} from 'react-native';
 
-export const getSizeStyle = (size: EButtonSize) => {
+export const getSizeStyle = (size: TButtonSize) => {
   let containerSizeStyle: StyleProp<ViewStyle> = {};
   let titleSizeStyle: StyleProp<TextStyle> = {};
```
Comment (lasharela) [inline src/components/shared/getSizeStyle.ts:6]: Those lines sounds very weird for me. something wrong here. Let's review later.

**Review by lasharela — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (digmel) [reply src/App.tsx:2]: got it

File: src/components/config/color.ts (hunk 2 above)
Comment (digmel) [reply src/components/config/color.ts:27]: no we have apple as a black color

File: src/components/header/Header.tsx (hunk 3 above)
Comment (digmel) [reply src/components/header/Header.tsx:47]: got it

File: src/components/label/Label.tsx (hunk 4 above)
Comment (digmel) [reply src/components/label/Label.tsx:47]: got it

File: src/components/shared/getSizeStyle.ts (hunk 5 above)
Comment (digmel) [reply src/components/shared/getSizeStyle.ts:6]: I know, `button` and `text input` component needs to modify, they all using this shared method `getSizeStyle`

**Pull request merged**